File: ruby/pangram/pangram.rb
class Pangram
  def self.is_pangram?(str)
    alphabet = "abcdefghijklmnopqrstuvwxyz".split("")
    count = 0
    alphabet.each { |letter|
      if str.downcase.match(letter) != nil
        count += 1
      end
    }

    count >= 26
  end
end

  module BookKeeping
    VERSION = 2 # Where the version number matches the one in the test.
  end

File: javascript/pangram/pangram.js
var Pangram = function(phrase) {
  this.phrase = phrase;
  this.alphabet = "abcdefghijklmnopqrstuvwxyz".split("");
};

Pangram.prototype.isPangram = function() {
  var phrase = this.phrase.toLowerCase();

  var letterCount = 0;
  this.alphabet.forEach(function(letter){
    if (phrase.match(letter) !== null){
      letterCount++;
    }
  });

  return (letterCount >= 26);

}

module.exports = Pangram;